tests/compiler/rt/instanceof.debug.wat

```diff
@@ -2805,6 +2742,39 @@
   i32.add
   global.set $~lib/memory/__stack_pointer
  )
+ (func $~instanceof|rt/instanceof/Cat (type $i32_=>_i32) (param $0 i32) (result i32)
+  (local $1 i32)
+  block $is_instance
+   local.get $0
+   i32.const 8
+   i32.sub
+   i32.load $0
+   i32.const 4
+   i32.eq
+   br_if $is_instance
+   local.get $0
+   call $~instanceof|rt/instanceof/BlackCat
+   br_if $is_instance
+   i32.const 0
+   return
+  end
+  i32.const 1
+ )
+ (func $~instanceof|rt/instanceof/BlackCat (type $i32_=>_i32) (param $0 i32) (result i32)
+  (local $1 i32)
+  block $is_instance
+   local.get $0
+   i32.const 8
+   i32.sub
+   i32.load $0
+   i32.const 5
+   i32.eq
+   br_if $is_instance
+   i32.const 0
+   return
+  end
+  i32.const 1
+ )
```

Comment:
An alternative here is to bake all possible ids into one function instead of calling through to extendees / implementers. Would benefit perf (possibly allowing `br_table`s), but duplicates checks for ids. Might be worth it.

Reply:
Decided to make this change